.gitignore

```diff
@@ -5,3 +5,5 @@
 /moz/web-ext-artifacts/
 go.mod
 go.sum
+/spec
+strings.go
```

Comment:
Please add a newline at the end of the file.

Reply:
ok i would